### DISCUSSION TITLE
Required input files for Landsat force-l2ps

### DISCUSSION THREAD

**Discussion by lukeroth** [davidfrantz/force#298] [Q&A] [answered]
Is there a list of the minimum set of input Landsat scene files required for process a Landsat scene through force-l2ps?  Aside from the imagery bands a number of extra files are typically present; however, we are having difficulty obtaining some of these extra files for a number of scenes, such as:
SAA.TIF, SZA.TIF, VAA.TIF, VZA.tif, QA_RADSAT.TIF, QA_PIXEL.TIF, MTL.xml, MTL.txt, ANG.txt
We would like to run as many of these scenes with partially missing data as possible.
Thanks!
Luke

**Comment by davidfrantz** [comment, marked as answer]
Hi @lukeroth,

sorry for the late answer, but actually only MTL.txt is needed. 

Best,
David

**Reply by lukeroth** [reply]
Thanks, David!
Luke
